.env.production

```diff
@@ -1 +1,3 @@
 NEXT_PUBLIC_BACKEND_URL=/api/mock
+GOOGLE_CLIENT_ID = '495274309991-utft6kl7jcdds3vecom87m1l9c8clf0e.apps.googleusercontent.com'
+GOOGLE_SECRET = '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'
```

Comment:
Never put sensitive information in the git history, especially considering that this is a public GitHub repo